controller.py

```diff
@@ -78,6 +84,10 @@ def annotate_feed(self, feed):
         feed.add_link_to_feed(
             feed.feed, href=search_url, rel="search", type=OPENSEARCH_MEDIA_TYPE
         )
+        register_url = self.app.url_for("register")
+        feed.add_link_to_feed(
+            feed.feed, href=register_url, rel="register"
+        )
```

Comment:
Let's go with register for now. I've asked Hadrien and Mike Amundsen if they know of something else we can use.